java/src/com/ibm/streamsx/topology/generator/spl/SPLGenerator.java

```diff
@@ -83,6 +98,406 @@ public String generateSPL(JsonObject graph) throws IOException {
         return sb.toString();
     }
     
+    private void separateIntoComposites(JsonObject graph){
+
+        compOperatorStarts.add(OpProperties.PARALLEL);
+        compStarts.add(BVirtualMarker.PARALLEL.kind());
+        compEnds.add(BVirtualMarker.END_PARALLEL.kind());
+        
+//        compOperatorStarts.add(null);
```

Comment:
Why the commented out code?

Reply:
OK - it's a TODO, useful to leave TODO comments for the review and as a reminder.